src/core/namespace.c

```diff
@@ -83,7 +83,6 @@ static const TargetMount protect_kernel_tunables_table[] = {
         { "/proc/sysrq-trigger",        READONLY,       true  },
         { "/proc/latency_stats",        READONLY,       true  },
         { "/proc/mtrr",                 READONLY,       true  },
-        { "/proc/apm",                  READONLY,       true  },
```

Comment:
This line that you removed is not about supporting APM. Quite the opposite — it's about blocking support for APM. It's the same as the `@obsolete` syscall list that we carry and the long list of useless old syscalls: because they are not used, it's all the more likely that there's some bug in them, and all the more worthwhile to block them. Version 3.3 that wikipedia mentions is irrelevant; instead we care about version 4.9 that still has CONFIG_APM to it's perfectly possible to have `/proc/apm` on a very modern kernel (proc(5) implies that the file is present no matter if APM is working or not).